Feat - Community add-on (#684)
sveltejs/cli: documentation/docs/20-commands/20-sv-add.md

- [`sveltekit-adapter`](sveltekit-adapter)
- [`tailwindcss`](tailwind)
- [`vitest`](vitest)

> ## Community add-ons

Reviewer: We should consider extracting this into a separate page, as most of this is relevant for `sv add` as well. 

Also, the docs about the `file:` protocol should be in the creating addons developer section, not in the user section.

Reviewer: Looks good now

> [!NOTE]
> Svelte maintainers have not reviewed community add-ons for malicious code!

You can find [community add-ons on npm](https://www.npmjs.com/search?q=keywords%3Asv-add) by searching for `keywords:sv-add`.

### How to install a community add-on

```sh
npx sv add [PROTOCOL][COMMUNITY_ADDON]
```

You can:

- mix and match official and community add-ons
- use the interactive prompt or give args to the cli
- use the `--add` option in the `create` command

```sh
npx sv add eslint @supacool
```

```sh
npx sv create --add eslint @supacool
```

### Protocols

We support two protocols for community add-ons:

- `@[ORG]` - for `sv` add-ons published under an npm organization.
  - Giving only the organization name will look for `@[ORG]/sv` _(preferred)_

>   - This `@[ORG]/my-super-cool-sv-addon` will also work, _but less nice!_

Reviewer: ```suggestion
  - This `@[ORG]/my-super-cool-sv-addon` will also work, _but is more wordy!_
```
